fetch.bs

```diff
@@ -5658,11 +5658,12 @@ however, it is perfectly fine to do so.
 
 <p>If <a>CORS protocol</a> requirements are more complicated than setting
 `<a http-header><code>Access-Control-Allow-Origin</code></a>` to <code>*</code> or a static
-<a for=/>origin</a>, `<code>Vary</code>` is to be used.
+<a for=/>origin</a>, `<code>Vary</code>` is to be used for <em>all</em> responses for a particular
+resource — not just for responses to <a lt="CORS request">CORS requests</a> for that resource.
 [[!HTML]]
 [[!HTTP]] [[!HTTP-SEMANTICS]] [[!HTTP-COND]] [[!HTTP-CACHING]] [[!HTTP-AUTH]]
 
-<pre id=example-vary-origin class=example>Vary: Origin</pre>
+<pre id=example-vary-origin class=example>Vary: Origin, Access-Control-Request-Headers, Access-Control-Request-Method</pre>
```

Comment:
Yeah, though we haven't written that down yet and last I heard Chrome doesn't believe in that. :/